apps/roam/src/styles/styles.css

```diff
@@ -114,6 +114,30 @@
   border-bottom: 1px solid rgba(16, 22, 26, 0.15);
 }
 
+.roamjs-create-relation-dialog-body {
+  background-color: #e8eff3;
+}
+
+.dg-detached-header {
+  background-color: #c0d8e6;
+  color: #2c5f95;
+  margin: 1em;
+  font-size: 16px;
+  min-height: 20px;
+  padding: 5px;
+}
+
+.dg-detached-header .bp3-icon {
+  color: #c0d8e6;
+  background-color: #2c5f95;
+  padding: 1px;
+  margin-left: 0.7ex;
+}
+
+.full-popover-width .bp3-popover-target {
```

Comment:
1) This looks like it is targeting some generic BP3 component, so it will likely target more than just what you want it to and have negative consequenses.  Let's try to make it more specific with a class or id of the component you are targeting.
2) What is this meant to target?  If the class or id isn't obvious, let's leave a breadcrumb in the comment to be clear.

Reply:
It's targeting the inside of the component. I created the `.full-popover-width` marker class precisely to make the target specific, without side effects.

Reply:
Let's make the class more specific.  `.full-popover-width` could be referring to a lot of things.